### ISSUE TITLE
 [Medium]: Chrome+Jaws : Missing list markup for controls.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#8509]
## Action Performed:
1. Using Chrome, open URL: staging.new.expensify.com
2. Navigate to Room Details Page by clicking on the header of the report page.
3. Navigate to the 'Members' control using the Tab key and activate it.
4. Navigate to the 'applausetester@applause.expensifail...', 'applausetester+ab1243@applause.expensifail...', 'Tana Clifford...', etc. controls.
5. Observe the screen reader announce list markup.

## Expected Result:
When list mark-up is provided for the mentioned content, screen reader users will identify a number of list items.

## Actual Result:
The list mark-up is not used to present related information on the screen. For instance, the related information such as 'applausetester@applause.expensifail...', 'applausetester+ab1243@applause.expensifail...', 'Tana Clifford...', etc. controls are not marked as list items that visually look like a list. As a result, screen reader users might find it difficult to navigate to the lists and different list items quickly.

## Workaround:
Yes

## Area issue was found in:
All pages of the Expensify application where such types of controls are appearing.

## Failed WCAG checkpoints
1.3.1

## User impact:
When lists do not use list markup, screen reader users will lose the semantic information the structure of the list provides visual users.

## Suggested resolution:
Use appropriate list mark-up, such as 'li' and 'ul' to identify related information. In addition, if required apply styles to maintain the visual presentation of the web page.

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
 Reference link: https://www.w3.org/WAI/tutorials/page-structure/content/
**Issue reported by:** Sumit

![#8008_Missing list markup for controls](https://user-images.githubusercontent.com/98817464/162001518-846eb395-e3a0-471f-9534-4f856a51ca2a.png)
